app/(auth)/auth/confirm/route.ts

```diff
@@ -7,7 +7,7 @@ export async function GET(request: NextRequest) {
   const { searchParams } = new URL(request.url)
   const token_hash = searchParams.get('token_hash')
   const type = searchParams.get('type') as EmailOtpType | null
-  const next = searchParams.get('next') ?? '/'
+  const next = searchParams.get('next') ?? '/settings'
```

Comment:
What does this do?

Reply:
It redirects the user to `/settings` on an successful **oauth** sign in